server/src/api/v1/users/update-user.ts

```diff
@@ -0,0 +1,110 @@
+import { Request, Response } from "express";
+import isBlank from "is-blank";
+import { checkKeysAreNotEmptyOrNotSet } from "../../../functions/check-inputs.func";
+import { mapUser } from "../../../functions/map-users.func";
+import { wrapResponse } from "../../../functions/response-wrapper";
+import { IncomingUser, InternalUser } from "../../../interfaces/users.interface";
+import { User } from "../../../models/user.model";
+import { Customer } from "../../../models/customer.models";
+import * as EmailValidator from 'email-validator';
+import { Vars } from "../../../vars";
+
+export async function updateUser(req: Request, res: Response) {
+    let success = true;
+    let user: User | null;
+    let customer: Customer | null;
```

Comment:
Wenn du hier den customer explizit mit type Customer | null angibst, dann meckert er unten in Zeile 62, da du customer ja ggf nicht setzt, wenn es nicht in die If-Bedingung in Zeile 43 reinspringt.